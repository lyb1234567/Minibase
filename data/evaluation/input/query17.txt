Q(x, y, z, SUM(y*y)) :- R(x, y, z), S(u, w, t)