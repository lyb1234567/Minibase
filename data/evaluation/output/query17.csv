1, 9, 'adbs', 405
2, 7, 'anlp', 245
4, 2, 'ids', 20
5, 0, 'sdp', 0
8, 1, 'mlpr', 5
8, 9, 'rl', 405
8, 9, 'ppls', 405
