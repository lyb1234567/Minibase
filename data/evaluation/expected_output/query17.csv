8, 9, 'ppls', 405
5, 0, 'sdp', 0
8, 1, 'mlpr', 5
1, 9, 'adbs', 405
2, 7, 'anlp', 245
8, 9, 'rl', 405
4, 2, 'ids', 20